### PR TITLE
REF: BIP47 under the hood

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4a4mqecbjwxhxrg6owx6bl2laq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1418)

File: class/wallets/abstract-hd-electrum-wallet.ts [hunk 1]
```diff
@@ -1532,6 +1537,60 @@ export class AbstractHDElectrumWallet extends AbstractHDWallet {
...
+            return pc; // found it!
+          }
+        }
+      }
+    }
+
+    // checking txs we sent to counterparties
+
+    for (const pc of this._send_payment_codes) {
+      for (const tx of this.getTransactions()) {
```
Comment (Overtorment) [inline class/wallets/abstract-hd-electrum-wallet.ts:1579]: oops i think i need tofilter by txid here

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @marcosrdz] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/marcosrdz

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/4kl5t2clbwm5vxpiuo246oppuu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1429)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @marcosrdz] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/marcosrdz

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Pull request merged**